src/content/development/security/_index.en.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Security Reporting"
```

Comment:
src/content/community/security/_index.en.md → ...content/development/security/_index.en.md 

IMO, reporting of security issues should be visible to anyone.. this sends the message that this could be done by developers only?

Reply:
To me it isn't less visible by putting it in this section, the community section isn't more visible than the development section. My thinking is that someone showing up to report an issue would consider that a development action, and look in the development section.

Reply:
Ok, I am fine either way.